platform/platform-migratable/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -22,5 +22,5 @@
     </bean>
 
```

Comment:
It doesn't look like you are injecting the new constructor parameters in your blueprint.